src/ui/reducers/layout.ts

```diff
@@ -2,12 +2,13 @@ import { sourcesDisplayed } from "devtools/client/debugger/src/reducers/ui";
 import { LayoutAction } from "ui/actions/layout";
 import { UIState } from "ui/state";
 import { LayoutState } from "ui/state/layout";
+import { features, prefs } from "ui/utils/prefs";
 
 export const syncInitialLayoutState: LayoutState = {
   consoleFilterDrawerExpanded: true,
   showCommandPalette: false,
   selectedPrimaryPanel: "events",
-  viewMode: "non-dev",
+  viewMode: features.defaultToDevMode ? "dev" : "non-dev",
```

Comment:
I wonder if this prop should just be a boolean too?

Reply:
I'm down for this.

Reply:
Actually, upon looking into this, it does not quite feel right to me. I think for the purposes of having a value that backs a UI checkbox, using a `boolean` is reasonable, but if we push this change down to the actual state, then we are making it harder to make a change that might add a third-view (not that this is an immediate concern, but just thinking of the future dev who might have to back these changes out if that happened).

Reply:
Oh, lol, I totally did not realize that Jason's comment was in response to the original comment here. I don't like that GH shows replies as separate comments when you are just looking at the "conversation" tab